src/main/frontend/src/components/WordForm.js

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from "react";
+
+function WordForm() {
+  const [languagesList, setLanguagesList] = useState([]);
+  useEffect(() => {
+    fetch("/api/v1/languages")
+    .then(response => response.json())
+    .then(response => {
+      setLanguagesList(response);
+    })
+  }, []);
+  const dropdown = languagesList.map((language) =>
+  <option key={language.id} value={language.name}>{language.name}</option>)
+  return (
```

Comment:
Can you fix the formatting here please?
Add an empty line between 13 & 14 and indent 16-41 properly